controllers/test/suite_e2e_test.go

```diff
@@ -15,7 +16,7 @@ import (
 func initTestConfig() *testConfigOptions {
 	return &testConfigOptions{
 		useExistingCluster: true,
-		crdVersion:         "v1beta1",
+		crdVersion:         "v1",
```

Comment:
does it means we are loosing `v1beta1` support?

Reply:
No, this only affects tests. The testing implementation we use no longer supports v1beta1 (it's been deprecated for a bit), but since we support both, we don't have a problem.